src/patternfly/components/NotificationDrawer/notification-drawer.scss

```diff
@@ -97,6 +134,49 @@
   --pf-c-notification-drawer__group-toggle-icon--Transition: .2s ease-in 0s;
   --pf-c-notification-drawer__group--m-expanded__group-toggle-icon--Rotate: 90deg;
 
+  // Task group toggle
+  --pf-c-notification-drawer__task-group-toggle--PaddingTop: var(--pf-global--spacer--md);
+  --pf-c-notification-drawer__task-group-toggle--PaddingRight: var(--pf-global--spacer--xs);
+  --pf-c-notification-drawer__task-group-toggle--PaddingBottom: var(--pf-global--spacer--md);
+  --pf-c-notification-drawer__task-group-toggle--PaddingLeft: var(--pf-global--spacer--md);
+  --pf-c-notification-drawer__task-group-toggle--BackgroundColor: var(--pf-global--BackgroundColor--100);
+  --pf-c-notification-drawer__task-group-toggle--BorderColor: var(--pf-global--BorderColor--100);
+  --pf-c-notification-drawer__task-group-toggle--BorderBottomWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-notification-drawer__task-group-toggle--OutlineOffset: #{pf-size-prem(-4px)};
+
+  // Task group toggle header
+  --pf-c-notification-drawer__task-group-toggle-header--AlignItems: baseline;
+
+  // Task group toggle title
+  --pf-c-notification-drawer__task-group-toggle-title--MarginRight: var(--pf-global--spacer--md);
+  --pf-c-notification-drawer__task-group-toggle-title--max-lines: 1;
+
+  // Task group toggle status
+  --pf-c-notification-drawer__task-list-group--m-info__task-group-toggle-status--Color: var(--pf-global--info-color--100);
+  --pf-c-notification-drawer__task-list-group--m-warning__task-group-toggle-status--Color: var(--pf-global--warning-color--100);
+  --pf-c-notification-drawer__task-list-group--m-danger__task-group-toggle-status--Color: var(--pf-global--danger-color--100);
+  --pf-c-notification-drawer__task-list-group--m-success__task-group-toggle-status--Color: var(--pf-global--success-color--100);
+  --pf-c-notification-drawer__task-list-group--m-default__task-group-toggle-status--Color: var(--pf-global--default-color--200);
+  --pf-c-notification-drawer__task-group-toggle-status--Color: var(--pf-c-notification-drawer__task-list-group--m-default__task-group-toggle-status--Color);
+  --pf-c-notification-drawer__task-group-toggle-status--MarginRight: var(--pf-global--spacer--md);
+
+  // Task group toggle action
+  --pf-c-notification-drawer__task-group-toggle-action--MarginRight: var(--pf-global--spacer--md);
+
+  // Task group toggle count
+  --pf-c-notification-drawer__task-group-toggle-count--MarginRight: var(--pf-global--spacer--md);
+
+  // Task group toggle icon
+  --pf-c-notification-drawer__task-group-toggle-icon--MarginRight: var(--pf-global--spacer--md);
+  --pf-c-notification-drawer__task-group-toggle-icon--Color: var(--pf-global--Color--200);
+  --pf-c-notification-drawer__task-group-toggle-icon--Transition: .2s ease-in 0s;
```

Comment:
Can we update this (and the notification item icon transition) to use `var(--pf-global--Transition)`? We've since started using that for these icon toggle transitions.